scripts/azureml-assets/azureml/assets/model/model_utils.py

```diff
@@ -151,3 +143,46 @@ def prepare_model(spec_path, model_config, registry_name, temp_dir):
     except Exception as e:
         logger.log_error(f"prepare model failed for {spec_path}. Error {e}")
         return None, False
+
+
+def update_model_metadata(
+    mlclient: MLClient,
+    model_name: str,
+    model_version: str,
+    update: AssetVersionUpdate,
+):
+    """Update the mutable metadata of already registered Model."""
+    try:
+        model = mlclient.models.get(name=model_name, version=model_version)
+
+        need_update = False
+        if update is not None:  # to update older version of models
```

Comment:
I think you can remove this line.

Reply:
yeah !!! this will always be a non null value for current flow

Reply:
Done